assets/stylesheets/layout/_sidebar.scss

```diff
@@ -126,9 +126,7 @@
 		overflow: hidden;
 		display: flex;
 
-		&:focus {
-			outline: none;
-		}
```

Comment:
Unfortunately we still need this for a few cases. There's a bit of an ugly outline in firefox after clicking a link if this is removed.
![ff-outline](https://cloud.githubusercontent.com/assets/1270189/12368797/cc74dcf0-bba3-11e5-9ce0-8f4b1cd70f42.png)